lib/src/cli.rs

```diff
@@ -348,6 +350,46 @@ enum ProgressOrFinish {
     Finished(Result<ostree_container::Import>),
 }
 
+/// Render an import progress notification as a string.
+pub fn layer_progress_format(p: &ImportProgress) -> String {
```

Comment:
I made this `pub` so we can re-use it in rpm-ostree, which wants to render status via dbus, not `println!`.